hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/DefaultContainerExecutor.java

```diff
@@ -98,6 +104,13 @@ public DefaultContainerExecutor() {
     }
   }
 
+  @VisibleForTesting
```

Comment:
Looks this constructor is only for Testing purpose. Shall we use setMethod instead which is followed in other classes.

Reply:
removed the constructor and added setter which is set with annotation VisibleForTesting